src/engines/gnucap.c

```diff
@@ -60,6 +60,9 @@ static struct analysis_tag analysis_tags[] = {
 #define TAGS_COUNT (sizeof(analysis_tags) / sizeof(struct analysis_tag))
 
 // Parser STATUS
+typedef enum { STATE_IDLE, IN_VARIABLES, IN_VALUES, STATE_ABORT } ParseDataState;
+static ParseDataState state;
+
```

Comment:
If you prefer, the above mentioned variable and the corresponding switch statement can be removed completely without loss of functionality:

https://github.com/gtrentalancia/oregano/commit/239026610c8bfb599e5a6bf5a5d998321ef84b50